test/test_browser.py

```diff
@@ -3027,6 +3027,10 @@ def test_glfw_events(self):
     self.btest('test_glfw_events.c', args=['-sUSE_GLFW=3', "-DUSE_GLFW=3", '-lglfw', '-lGL'], expected='1')
 
   @no_wasm64('SDL2 + wasm64')
```

Comment:
You need to move this decorator back down so it applies to `test_sdl2_image`

Reply:
I guess it is the github automatic merge that screwed it up :(